app/src/main/java/com/teambeme/beme/mypage/viewmodel/MyPageViewModel.kt

```diff
@@ -38,8 +38,29 @@ class MyPageViewModel @Inject constructor(
     val scrapScrollUp: LiveData<Unit>
         get() = _scrapScrollUp
 
-    private var page = 1
-    private var scrapPage = 1
+    private var _page: Int = 2
+    val page: Int
+        get() = _page
+
+    private var _tempPage: Int = 1
+    val tempPage: Int
+        get() = _tempPage
```

Comment:
```suggestion
    var tempPage: Int = 1
           private set
```

page말고 이부분을 이렇게 쓰려구!

Reply:
좋아요!!